src/public/report/styles.css

```
@@ -0,0 +1,351 @@
+.rp-dropzone small {
+  font-size: 0.78rem;
+  color: #9ca3af;
+}
+
+.rp-file-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  opacity: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)